_posts/2020-06-20-snap-minis.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Snap Minis: How we can help you bring your app into SnapChat"
+date: 2020-07-10 11:00:00
+reviewed: 2020-07-10 11:00:00
+categories: []
+author: ABizzinotto
+---
+
+Last month, [Snap](https://www.snap.com) announced a new product during the [Snap Partner Summit 2020](https://www.youtube.com/watch?v=BfvubHha69k), [Snap Minis](https://minis.snapchat.com/). Minis are bite-sized third-party applications that live inside [Snapchat](https://www.snapchat.com/) and can be accessed through Snapchat's Chat interface. They are built on the same principle and technology as the Snap Games and are easy to develop, allowing you to have an app that takes advantage of the existing [Snapchat](https://www.snapchat.com/) application and is available to Snapchat users worldwide.
+
+OmbuLabs recently developed a Snap Mini application for one of our clients. In this article I'll share with you the main advantages of launching a Snap Mini and how we can help you get in on this.
+
+<!--more-->
+
+First of all, why create a Snap Mini? Here are the main advantages of having your own app inside [Snapchat](https://www.snapchat.com/):
+
+## Easy to Develop
+Snap Minis exist inside the main Snapchat application. They are built on HTML5 and take advantage of the existing Snapchat app, which means they work cross-platform and are readily available to all Snapchat users, reducing the scope of the project.
+
+Additionally, Minis work on both Android and iOS devices. You do not need two different applications to cater to different users.
+
+## Easy to Use
+The Minis are lightweight applications easily accessible through the Chat interface of the Snapchat app. Users do not need to install a separate app and switch applications to use your service. They also don't need to wait for a download to finish.
+
+Snapchat users can launch a Snap Mini from within a conversation they're having with their friends. You can quickly collaborate or play with your friends, without having to open a new app or website.
+
+## Easy to Share with Friends
+A Mini can interact with both the chat and camera interfaces of the Snapchat application. This makes it easy for your users to share that they've used your mini with their friends.
+
+Their friends can then tap the chat card or camera sticker and be taken to your mini. All without leaving Snapchat.
+
+# What Can I do with Minis?
+
+There are several possibilities here. Minis can integrate with the Snapchat features and perform a variety of actions.
+
+Having in mind the concept of a Mini, a bite-sized application inside the Chat interface, you can define an actionable use case within your product and turn it into a mini. Some examples shared by Snap during the Snap Partner Summit 2020 include deciding what to watch, buying tickets, voter registration and health and fitness. If you have a specific action that can be performed by your audience, you can turn it into a Mini.
```

Comment:
Hmmm, this sentence sounds awkward. Maybe "If you keep the concept of a Mini in mind, that it is a bite-sized application inside the Chat interface, you can define an actionable use case within your product  and turn it into a Mini. "

or 

"Keeping in mind that a mini is a bite-sized application inside the Chat interface, you can define an actionable use case within your product and turn it into a mini."